tests/conftest.py

```diff
@@ -128,20 +128,26 @@ def _test_images_s3_bucket():
 MICROVM_S3_FETCHER = MicrovmImageS3Fetcher(_test_images_s3_bucket())
 
 
-def init_microvm(root_path, bin_cloner_path):
+def init_microvm(root_path, bin_cloner_path,
+                 fc_binary=None, jailer_binary=None):
     """Auxiliary function for instantiating a microvm and setting it up."""
     # pylint: disable=redefined-outer-name
     # The fixture pattern causes a pylint false positive for that rule.
     microvm_id = str(uuid.uuid4())
-    fc_binary, jailer_binary = build_tools.get_firecracker_binaries()
+
+    if fc_binary is None:
```

Comment:
Shouldn't we also check for `jailer_binary` being `None`? 

If `jailer_binary` is `None` and `fc_binary` not `None` then it would fail at line `143`.

I think that we can either split the `get_firecracker_binaries` function into separate functions for each binary or do the following:

```
default_fc_binary, default_jailer_binary = build_tools.get_firecracker_binaries()

if not fc_binary:
    fc_binary = default_fc_binary

if not jailer_binary:
    jailer_binary = default_jailer_binary
```

Reply:
I spent some time on this here. The idea was to avoid building the fc binaries when caller provided the fc_binary. The assert should point to the obvious error that the jailer is missing. Also, the caller should always provide both binaries to avoid using mixed binaries.